EXT_PRAYTRACE.md

```diff
@@ -0,0 +1,258 @@
+Domain-Specific Naming Conventions for Photon Raytracing Codes
+===========================================================================
+
+openPMD extension name: `PRAYTRACE`
+
+
+Introduction
+------------
+
+This extension to the openPMD addresses the domain-specific requirements
+met in raytracing codes (e.g. OASYS). Each ray has the information stored
+in its own record, e.g. data/1/particles/rays/0...N/positions and therefore
+has to have the same length.
+
+Particle Based Records (Fields)
+---------------------------
+
+### Root attributes
+
+- `openPMD`: `0.0.2`
+- `openPMDextension`: `PRAYTRACE`
+- `author`: `Aljosa Hafner <aljosa.hafner@ceric-eric.eu>`
+- `date`: `10.10.2019 00:00:00 +0100`
+- `nRays`
+	- type: *(uintX)*
+	- description: Number of initialised rays
+	- scope: *required*
+	- unitSI: 1.0
+	- unitDimension: [unitless] (0., 0., 0., 0., 0., 0., 0.)
+
+### Structure
+data/1/particlesPath/rays/0...N/position...
```

Comment:
if ray numbering is 0 based, last should be N-1, right?

Reply:
Hey, I agree, I'll correct it. Regarding the hierarchy, I followed the example from NRAYTRACING, where the attributes are also marked like that: 'position/' - 'position/x'